File: Implemetation, Math/Java/4948 베르트랑 공준.java
import java.util.*;

public class Main {
    
    public static boolean isPrime(int number) {
        if (number % 2 == 0) return false;
        for(int i = 3; i <= (int)Math.pow(number, 0.5); i+=2) {
            if(number % i == 0) return false;
        }
        return true;
    }
    
    public static void main(String args[]) {
        Scanner sc = new Scanner(System.in);
        
        while(true) {
            int n = sc.nextInt();
            if(n == 1) {
                System.out.println(1); continue;
            }
            if(n == 0) break;
            int count = 0;
            for(int i = n+1; i <= 2*n; i++) {if(isPrime(i)) count++;}
            
            System.out.println(count);
        }

    }
}

File: Implemetation, Math/Kotlin/4948 베르트랑 공준.kt
import java.util.*;
import kotlin.math.*;

fun isPrime(number : Int) : Boolean {
    if(number % 2 == 0) return false
    for(i in 3..Math.pow(number.toDouble(), 0.5).toInt()) {
        if(number % i == 0) return false
    }
    return true

}

fun main(args: Array<String>) {
    var sc = Scanner(System.`in`)
    while(true) {
        var n = sc.nextInt();
        if(n == 1) {println(1); continue;}
        if(n == 0) break;
        var count = 0;
        for(i in n+1..(2*n)) {if(isPrime(i)) count++}
        println(count)
    }
    
}
